ddtrace/tracer/tracer_test.go

```diff
@@ -464,6 +464,19 @@ func TestTracerNoDebugStack(t *testing.T) {
 	assert.Empty(s.Meta[ext.ErrorStack])
 }
 
+func TestTracerNoDebugStackFinishSetTag(t *testing.T) {
```

Comment:
Instead of creating a separate test, create two sub-tests inside `TestTracerNoDebugStack` please.